students/gangho/users/views.py

```diff
@@ -0,0 +1,44 @@
+import json
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from users.validators import validate_email, validate_password
+
+
+class SignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if validate_email(data['email']) is None:
```

Comment:
파이썬 PEP 8 프로그래밍 권고에는 비교문 관련해서 어떤 형식을 사용하라고 안내되어 있습니다.

- True/False를 비교할 때는 if문에 변수만 사용하도록 한다.
- 동등성(단순히 값만 비교하는 경우)을 검증할 때는 `==` 혹은 `!=` 같은 연산자를 사용하도록 한다.
- 동일성(객체의 고유 메모리 주소로 비교하는 경우)을 검증할 때는 is나 is not을 사용한다.

즉 아래의 경우는 True/False만을 비교하기 때문에 `==`는 사용할 필요가 없습니다.

```python
if not ...:
```
```suggestion
            if not validate_email(data['email']):
```

Reply:
View에서 작성한 validate를 validator.py로 옮기고, validator.py에서 raise validationerror 일으키도록 수정하였습니다!